### PR TITLE
Use externally-valid include again

### PR DESCRIPTION
**Pull request by flub**
This switches back to using #include <deltachat/deltachat.h> which is
the way the header should be included for users of libdeltachat.  This
needed
https://github.com/deltachat/deltachat-core/commit/12ef73c8e76185f9b78e844ea673025f56a959ab
which was not yet committed when this was first made.

It allows us to include things the correct way both when compiling
against an installed and against an un-installed library without
having to abuse the -I flags.

### CODE REVIEW

**Comment by flub** [comment]
Hum, it seems the failure later passes in the next build so I think it can be ignored.

**Comment by flub** [comment]
fixes https://github.com/deltachat/deltachat-node/issues/286

**Comment by flub** [comment]
I think the submodule needs to be updated to point to the most recent master on deltachat-core too before merging this.  Not sure how to do that.

**Comment by flub** [comment]
Seems to work now.  The "push" travis config seems to miss an environment var so is not usable in it's current form?

File: scripts/ci-build.sh [hunk 1]
```diff
@@ -37,6 +38,4 @@ else
 fi
 
 export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
-$EXEC pkg-config --cflags deltachat
-$EXEC pkg-config --libs deltachat
-$EXEC npm install;
+$EXEC npm install --dc-system-lib=$SYS_DC_CORE;
```
Comment (ralphtheninja) [inline scripts/ci-build.sh]: Where is this `--dc-system-lib` going?

Comment (flub) [reply scripts/ci-build.sh]: It's handled by `scripts/npm-install.js` to either invoke `npm run rebuild-all` or invoke `node-gyp rebuild -- -Dsystem_dc_core=true` when calling `npm install`.

Comment (ralphtheninja) [reply scripts/ci-build.sh]: @flub Nevermind, found it :)

File: binding.gyp [hunk 2]
```diff
@@ -42,8 +42,8 @@
                                     "-Wl,-rpath='$$ORIGIN/../../<(submod_builddir)/src'",
                                 ],
                             }, { # OS == 'mac'
-                                "ldflags": [
-                                    "-Wl,-rpath='@loader_path/../../<(submod_builddir)/src'",
+                                "libraries": [
+                                    "-rpath '@loader_path/../../<(submod_builddir)/src'",
```
Comment (ralphtheninja) [inline binding.gyp]: :+1:

**Comment by Jikstra** [comment]
Sehr schön!

**Pull request merged**